### PR TITLE
fix(contextualhelp): dark mode font color

### PR DESCRIPTION
**Pull request by mlogsdon18**
## Description

Adds `.spectrum-Heading` and `.spectrum-Body` classes to the heading and paragraph tags in order to get the global font colors needed since font color was not a specified token for this component. 

## How and where has this been tested?
 - Latest versions of Chrome, Safari, Firefox

## To-do list

- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).
- [x] I have tested these changes in Windows High Contrast mode.
- [x] I have updated any relevant storybook stories and templates. 

- [x] This pull request is ready to merge.

### CODE REVIEW

**Review by Rajdeepc — APPROVED** [review]

File: components/contextualhelp/metadata/contextualhelp.yml [hunk 1]
```diff
@@ -18,17 +18,17 @@ examples:
               </svg>
             </button>
             <div class="spectrum-Popover is-open spectrum-Popover--bottom-start spectrum-ContextualHelp-popover" role="presentation">
-              <h2 class="spectrum-ContextualHelp-heading">Permission required</h2>
-              <p class="spectrum-ContextualHelp-body">Cupcake ipsum dolor sit amet jelly beans. Chocolate jelly caramels. Icing soufflé chupa chups donut cheesecake. Jelly-o chocolate cake sweet roll cake danish candy biscuit halvah.</p>
+              <h2 class="spectrum-Heading spectrum-Heading--sizeM spectrum-ContextualHelp-heading">Permission required</h2>
```
Comment (castastrophe) [inline components/contextualhelp/metadata/contextualhelp.yml:21]: I think ideally we'd simplify this such that the spectrum heading and body classes aren't needed and can be extended in the contextual help class. The more isolated a component is from other dependencies, the easier it is to convert to a web component.

**Review by castastrophe — APPROVED** [review]

File: components/contextualhelp/metadata/contextualhelp.yml (hunk 1 above)
Comment (mlogsdon18) [reply components/contextualhelp/metadata/contextualhelp.yml:21]: Right. I can go ahead and just use the tokens that the heading and body classes use even though they weren't specified for this component. Let me see about making that change!

**Comment by mlogsdon18** [comment]
@castastrophe I was able to add in the tokens so that we could remove the `.spectrum-Heading` and `.spectrum-Body` classes. I also noticed that two of the tokens I was originally missing have since been added so I updated those as well

**Review by pfulton — CHANGES_REQUESTED** [review]
Thanks for taking care of this, @mlogsdon18. Looks like this resolves the font color issue.

I do have one more thing related to this component: it looks like the button's top edge-to-icon spacing is off
Screen Shot 2023-05-02 at 10 42 39 AM
Screen Shot 2023-05-02 at 10 42 50 AM


Would you mind looking into that?

File: components/contextualhelp/stories/template.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ export const Template = ({
       })}
       ${Popover({
         content: [
-          title ? html`<h2 class="${rootClass}-heading">${title}</h2>` : "",
-          body ? html`<p class="${rootClass}-body">${body}</p>` : "",
+          title ? html`<h2 class="spectrum-Heading ${rootClass}-heading">${title}</h2>` : "",
+          body ? html`<p class="spectrum-Body ${rootClass}-body">${body}</p>` : "",
```
Comment (castastrophe) [inline components/contextualhelp/stories/template.js:41]: Are the header and body classes still needed?

**Review by pfulton — APPROVED** [review]

**Pull request merged**